packages/xo-web/src/xo-app/disk-import/index.js

```diff
@@ -108,12 +108,16 @@ const DiskImport = decorate([
         <Row>
           <LabelCol>{_('importToSr')}</LabelCol>
           <InputCol>
-            <SelectSr onChange={effects.onChangeSr} required value={sr} />
+            <SelectSr onChange={effects.onChangeSr} required value={sr} allowIsoSr />
```

Comment:
Why don't you pass `predicate={isSrWritableOrIso}` instead of handling a rare case in the component itself?